.github/release-drafter-config.yml

```diff
@@ -5,16 +5,26 @@ categories:
     labels:
       - 'feature'
       - 'enhancement'
+      - 'c:feature'
+      - 'c:enhancement'
   - title: 'Bug Fixes'
     labels:
       - 'fix'
       - 'bugfix'
       - 'bug'
+      - 'c:bug'
+  - title: 'Breaking Changes'
+    labels:
+      - 'pr:break'
+  - title: 'Documentation'
```

Comment:
We use maintenance elsewhere.
The current big 4 are:
- feature
- bug
- maintenance
- breaking change

Reply:
I also added a fifth pillar `documentation`, so it could be easily left outside after reviewing it
The rest is indeed `maintenance` which we did not specifically name it till now.